src/components/TabBar.tsx

```diff
@@ -7,42 +7,50 @@ import { ThemeProps } from './theme';
 export interface TabProps extends ThemeProps, StyleProps {
   description: TabBar.TabDescription;
   selected?: boolean;
+  vertical?: boolean;
   onClick: (event: React.MouseEvent<HTMLDivElement>) => void;
 }
 
-const TabContainer = styled('div', (props: ThemeProps & { selected: boolean }) => ({
+const TabContainer = styled('div', (props: ThemeProps & { selected: boolean, $vertical: boolean }) => ({
   flex: '1 1',
   borderRight: `1px solid ${props.theme.borderColor}`,
+  borderBottom: (props.$vertical) ? `1px solid ${props.theme.borderColor}` : null,
   backgroundColor: props.selected ? `rgba(255, 255, 255, 0.1)` : `rgba(0, 0, 0, 0.1)`,
   opacity: props.selected ? 1 : 0.2,
   transition: 'background-color 0.2s, opacity 0.2s',
   padding: `${props.theme.itemPadding * 2}px`,
   ':last-child': {
-    borderRight: 'none'
+    borderRight: 'none',
+    borderBottom: 'none'
   },
   textAlign: 'center',
   userSelect: 'none',
   cursor: props.selected ? 'auto' : 'pointer',
+  position: 'relative',
+  minWidth: '45px',
 }));
 
 const TabIcon = styled(Fa, {
   paddingRight: '5px'
 });
 
-const TabText = styled('div', {
-  // transform: (props.isVertical) ? 'rotate(-90deg)' : null,
-	// transformOrigin: (props.isVertical) ? 'center' : null,
+const TabText = styled('div',  (props: {$vertical: boolean}) => ({
+  position: 'absolute', 
+  left: '50%', 
+  top: '50%',
+  transform: (props.$vertical) ? 'translate(-50%, -50%) rotate(-90deg)' : null,
```

Comment:
The `translate` should apply whether or not the text is vertical.

Reply:
Should be fixed now, good call